src/App.tsx

```diff
@@ -1,24 +1,164 @@
-/* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
-import { UserWarning } from './UserWarning';
+import React, { useEffect, useMemo, useState } from 'react';
+import classNames from 'classnames';
 
-const USER_ID = 0;
+import { Todo } from './types/Todo';
+import { Completion } from './types/Completion';
+import { TodoError } from './types/TodoError';
+import { USER_ID } from './constants';
+import {
+  deleteTodoFromServer,
+  getTodosFromServer,
+  updateTodoOnServer,
+} from './api/todos';
+
+import { TodoForm } from './components/TodoForm';
+import { TodoList } from './components/TodoList';
+import { TodoFilter } from './components/TodoFilter';
+import { NotificationError } from './components/NotificationError';
```

Comment:
Just a piece of advice - if you're not using separate folders for components, you can create an index file and export these elements directly from it)